### PR TITLE
Simplify class name parser in `SheetParser`

### PR DESCRIPTION
**Pull request by NduatiK**
@bcardarella, this PR strips out the excess code from #8

### CODE REVIEW

File: lib/live_view_native/stylesheet/sheet_parser/helper_functions.ex [hunk 1]
```diff
@@ -1,17 +0,0 @@
-defmodule LiveViewNative.Stylesheet.SheetParser.HelperFunctions do
```
Comment (bcardarella) [inline lib/live_view_native/stylesheet/sheet_parser/helper_functions.ex:1]: Let's keep this file as these helpers are not platform specific and will be shared by other client rules implementations. I can add a test for this specific file in this lib then in the client libs the rule parsers will import these helpers.

Comment (NduatiK) [reply lib/live_view_native/stylesheet/sheet_parser/helper_functions.ex]: Done

Comment (bcardarella) [reply lib/live_view_native/stylesheet/sheet_parser/helper_functions.ex]: Thank you! 💯

Comment (NduatiK) [reply lib/live_view_native/stylesheet/sheet_parser/helper_functions.ex]: What do helper functions receive? Variables?

Comment (bcardarella) [reply lib/live_view_native/stylesheet/sheet_parser/helper_functions.ex]: for now we're just supporting variables. They're intended to allow for the manipulation of the variable captured during pattern matching in the class name itself. Those values area always captured as a `String` and coercing to another value will be necessary

Comment (bcardarella) [reply lib/live_view_native/stylesheet/sheet_parser/helper_functions.ex]: @NduatiK whoops, missed your mention of updates, why don't you just make that a separate PR if still necessary

Comment (NduatiK) [reply lib/live_view_native/stylesheet/sheet_parser/helper_functions.ex]: #11

**Pull request merged**